src/main/java/core/basesyntax/CustomMathPow.java

```diff
@@ -9,6 +9,10 @@
      *      exponential - значение степени
      */
     public int pow(int number, int exponential) {
-        return 0;
+        double result = 1;
```

Comment:
Why you need double here?

Reply:
The test was failing with number = 9 and exponential = 10 without double.